versioned_docs/version-7.x/function-after-focusing-screen.md

```diff
@@ -51,26 +135,101 @@ React Navigation provides a [hook](https://reactjs.org/docs/hooks-intro.html) th
 
 This is particularly handy when we are trying to stop something when the page is unfocused, like stopping a video or audio file from playing, or stopping the tracking of a user's location.
 
-<samp id="simple-focus-effect" />
+<Tabs groupId="config" queryString="config">
+<TabItem value="static" label="Static" default>
+
+```js name="useFocusEffect hook" snack version=7
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  useFocusEffect,
+  createStaticNavigation,
+} from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+
+// codeblock-focus-start
+function ProfileScreen() {
+  useFocusEffect(
+    React.useCallback(() => {
+      alert('Screen was focused');
+      // Do something when the screen is focused
+      return () => {
+        alert('Screen was unfocused');
+        // Do something when the screen is unfocused
+        // Useful for cleanup functions
+      };
+    }, [])
```

Comment:
Probably its not needed because it is not used inside of hook - but i only have ESLint for proof 
<img width="620" alt="image" src="https://github.com/react-navigation/react-navigation.github.io/assets/59940332/c334a25f-b2e1-4bf3-bbde-0200d1df7a5a">